embulk-input-mysql/src/main/java/org/embulk/input/mysql/MySQLInputPlugin.java

```diff
@@ -156,7 +156,10 @@ private void loadTimeZoneMappings()
         // from the classloader. It seems like a bug of JDBC Driver where it should use the class loader
         // that loaded com.mysql.jdbc.TimeUtil class rather than system class loader to read the
         // property file because the file should be in the same classpath with the class.
-        // Here implements a workaround as as workaround.
+        // Here implements a workaround as a workaround.
+        //
+        // It's not 100% sure, but this workaround is necessary for Connector/J 5.x (com.mysql.jdbc.TimeUtil)
+        // only.
```

Comment:
```suggestion
        // This workaround seems required only for Connector/J 5.x (com.mysql.jdbc.TimeUtil),
        // not necessary for Connector/J 8.x (com.mysql.cj.util.TimeUtil).
        // TODO: Clarify for Connector/J 8.x just in case.
```

Reply:
Thanks fixed!